src/ft2font_wrapper.cpp

```diff
@@ -1597,33 +1590,7 @@ PyMODINIT_FUNC PyInit_ft2font(void)
         PyModule_AddIntConstant(m, "LOAD_TARGET_MONO", (unsigned long)FT_LOAD_TARGET_MONO) ||
         PyModule_AddIntConstant(m, "LOAD_TARGET_LCD", (unsigned long)FT_LOAD_TARGET_LCD) ||
         PyModule_AddIntConstant(m, "LOAD_TARGET_LCD_V", (unsigned long)FT_LOAD_TARGET_LCD_V)) {
-        Py_DECREF(m);
-        return NULL;
-    }
-
-    // initialize library
-    int error = FT_Init_FreeType(&_ft2Library);
-
-    if (error) {
-        PyErr_SetString(PyExc_RuntimeError, "Could not initialize the freetype2 library");
-        Py_DECREF(m);
-        return NULL;
-    }
-
-    {
-        FT_Int major, minor, patch;
-        char version_string[64];
-
-        FT_Library_Version(_ft2Library, &major, &minor, &patch);
-        sprintf(version_string, "%d.%d.%d", major, minor, patch);
-        if (PyModule_AddStringConstant(m, "__freetype_version__", version_string)) {
-            Py_DECREF(m);
-            return NULL;
-        }
-    }
-
-    if (PyModule_AddStringConstant(m, "__freetype_build_type__", STRINGIFY(FREETYPE_BUILD_TYPE))) {
-        Py_DECREF(m);
+        Py_XDECREF(m);
         return NULL;
```

Comment:
Should you de-init FreeType on failure?

Reply:
It wasn't done before but sure, that's a good idea.